Maestro/subscriptions.json

```diff
@@ -2064,10 +2064,11 @@
       "triggerPaths": [
         "https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/**/*",
         "https://github.com/dotnet/corefx/blob/release/2.1/src/Common/src/CoreLib/**/*",
+        "https://github.com/dotnet/corefx/blob/release/3.0/src/Common/src/CoreLib/**/*",
         "https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/shared/**/*",
         "https://github.com/dotnet/coreclr/blob/release/2.1/src/System.Private.CoreLib/shared/**/*",
-        "https://github.com/dotnet/corert/blob/master/src/System.Private.CoreLib/shared/**/*",
-        "https://github.com/dotnet/corert/blob/release/2.1/src/System.Private.CoreLib/shared/**/*"
```

Comment:
Why are we removing this line?

`https://github.com/dotnet/corert/blob/release/2.1/src/System.Private.CoreLib/shared/**/*`

Reply:
Oh, I just read why in your original description.

Reply:
> Note, you don't need to redeploy Maestro to make updates to the subscriptions.json file. Merging a PR that changes the master branch will automatically be picked up by the Maestro service.

Thanks.

> Why are we removing this line?

The release branches are not present in corert repo. so this link was just broken